masterlist.yaml

```
@@ -524,6 +524,22 @@ plugins:
       - 'Graphic Herbalism TR.ESP'
       - 'Graphic Herbalism TR Extra.ESP'
 
-#Any dynamic patches (not sure if multipatch is the  only one)
+  #Any dynamic patches
```

Kind of related, but we should match section name and formatting from the other masterlists.

```suggestion
###### Modding Tools - Generated Files ######
```